sdk/src/driftClient.ts

```diff
@@ -129,6 +130,8 @@ export class DriftClient {
 	authority: PublicKey;
 	marketLookupTable: PublicKey;
 	lookupTableAccount: AddressLookupTableAccount;
+	includeDelegates?: boolean;
+	authoritySubaccountMap?: [PublicKey, number][];
```

Comment:
sorry last nit pit, should this be a map instead of an array of arrays? if you wanted multiple sub accounts from same pub key, you'd have to do something like:
```
[
  [Pubkey.default(), 0],
  [Pubkey.default(), 1],
  [Pubkey.default(), 2],
]
```
with a map it'd be:
```
new Map([
  [Pubkey.default(), [0, 1, 2]]
])
```

(I think it'd be to be Map<string, number[]> since PublicKey doesnt really work as a key)